From 12bf719ab9a98571555ebe5752b672f9f7fbf56a Mon Sep 17 00:00:00 2001
From: Raman Pantsialeyeu <romasdk7@gmail.com>
Date: Wed, 25 Dec 2019 22:05:16 +0300
Subject: [PATCH] fixed 1.txt file after failed patch

---
 1.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/1.txt b/1.txt
index 6e3271f..a04210a 100644
--- a/1.txt
+++ b/1.txt
@@ -1,5 +1,5 @@
-modified two
-lines via git_0
+1
+1
 1
 1
 1
-- 
2.21.0.windows.1

